constraints.txt

```diff
@@ -13,7 +13,7 @@ backcall==0.2.0
 bcp47==0.0.4
 beautifulsoup4==4.12.2
 bidict==0.22.1
-bioregistry==0.6.99
+bioregistry==0.9.15
```

Comment:
Thanks to the LinkML folks for having fixed their `curies` dependency, we are no longer forcefully restricted to an old `bioregistry` version. 🎉

Reply:
🎉